frontend/packages/client/src/components/Proposal/ShareProposalDropdown.tsx

```diff
@@ -0,0 +1,39 @@
+import { ShareDropdown } from 'components';
+import { FRONTEND_URL } from 'const';
+import { Box } from '@chakra-ui/react';
+
+interface ShareProposalDropdownProps {
+  communityId: string;
+  proposalId: string;
+  proposalName: string;
+  userVoted: boolean;
+}
+const ShareProposalDropdown: React.FC<ShareProposalDropdownProps> = (
+  {
+    communityId,
+    proposalId,
+    proposalName = '',
+    userVoted,
+  } = {} as ShareProposalDropdownProps
+) => {
+  const proposalUrl = `${FRONTEND_URL}/#/community/${communityId}/proposal/${proposalId}`;
+
+  const twitterPost = `https://twitter.com/intent/tweet?text=${encodeURIComponent(
+    userVoted
+      ? `I just voted on ${proposalName} on CAST! ${proposalUrl}`
+      : `Check out ${proposalName} on CAST! ${proposalUrl}`
+  )} `;
```

Comment:
This should just be
```
const twitterPost = userVoted
      ? `I just voted on ${proposalName} on CAST! ${proposalUrl}`
      : `Check out ${proposalName} on CAST! ${proposalUrl}`
```

because below in the component you using the twitter intent url again